lib/ContextModule.js

```diff
@@ -394,6 +398,10 @@ class ContextModule extends Module {
 		this.buildInfo = {
 			snapshot: undefined
 		};
+		this.sourceOptions = {
+			context: options.context,
+			cacheObj: compilation.compiler.root
+		};
```

Comment:
```suggestion
```

This would need to be serialized to be able to pass it from `build` to `codeGeneration`, but the `compilation.compiler.root` object is not serializable, so that's not possible here.